canopy_readnml.F90

```diff
@@ -82,7 +77,7 @@ SUBROUTINE canopy_readnml (nlat,nlon,modlays,modres,href,z0ghc,lamdars, &
 
 !-------------------------------------------------------------------------------
 ! Set default real value for canopy vertical resolution (m) (Default = 0.5 m)
-    modres = 0.5_rk
+    modres = 0.5
```

Comment:
@zmoon From my understanding, the _rks are not needed here anymore and are redundant (causes warning in the debug compile), since these namelist variables are defined as real(rk) now in canopy_coord_mod.F90 ad canopy_vars_mod.F90.

Reply:
It seems the error was because it was defined in those multiple modules that were being used, so it was an "ambiguous reference". With your change it should be ok to `use canopy_const_mod, only: rk` and put them back.

Just because it is declared as `real(rk)` doesn't mean this doesn't have an effect, here is an example:
```fortran
program asdf
  use iso_fortran_env, only: rk => real64
  implicit none

  real(rk) :: x

  print *, 0.123
  print *, 0.123_rk

  x = 0.123
  print *, x
  x = 0.123_rk
  print *, x

end program
```
```
  0.123000003
  0.12300000000000000
  0.12300000339746475
  0.12300000000000000
```
Note the third one, from `x = 0.123`.

Reply:
@zmoon Thank you.  I will put them back in those modules that had the ambiguous references.

Reply:
@zmoon I have added them back in for the next [commit](https://github.com/noaa-oar-arl/canopy-app/pull/30/commits/c8c8bd95bc88bfa3ef6102c1551a48757536b46c).   I think this was the only place I removed them, but if you see others I appreciate it!  Thanks!

Reply:
If I notice others I will comment, but will now resolve this one. Thanks!